interface/new/new_comprehensive.php

```diff
@@ -381,13 +381,20 @@ function validate(f) {
         }
         <?php } ?>
         var msg = "";
+        var ins_error = '0';
         msg += "<?php echo htmlspecialchars(xl('The following fields are required'),ENT_QUOTES); ?>:\n\n";
         for ( var i = 0; i < errMsgs.length; i++ ) {
                msg += errMsgs[i] + "\n";
         }
+        <?php if($GLOBALS['primary_insurance_required']){ ?>
+          if($('#i1provider option:selected').val() == '') {
+             var ins_error = '1';
+             msg += "<?php echo htmlspecialchars(xl('Primary Insurance Provider Needed.'),ENT_QUOTES); ?> \n";
```

Comment:
"Primary Insurance Provider Entry Required."
or 
"Please enter Primary Insurance Provider Information"

Reply:
Which one would be best?

Reply:
I like the short one...because it is short.

Reply:
If you like yours, then we will just merge this.
Ain't merged anything yet today!